src/panels/ConfigPanel/ConfigPanel.tsx

```diff
@@ -300,6 +381,25 @@ function generateConfigElementList(
   return elements;
 }
 
+function generateChangedKeyValues(config: Config) {
+  const changedValues = Object.keys(config).reduce(
+    (acc: ReactNode[], key: string) => {
```

Comment:
It's unclear to me what `acc` means here? Can you use a clearer name?

Reply:
ahh, yes in the `reduce` call it's the `accumulator`. I can expand the variable name.